index.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>youtube</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav class="navbar">
+     <div class="header">
+         <a href="#">
+             <span class="icon">
+                <ion-icon name="logo-youtube"></ion-icon>
+             </span>
+             <span class="icon2">
+                 <h2>Youtube</h2>
+             </span>
+             <span class="icon3">
+                 <p>KE</p>
+             </span>
+         </a>
+     </div>
```

Comment:
Please use unordered list tags in your header codes, instead of using span tags

Reply:
- [ ] Nice review, however you need to indicate why you suggested the changes requested. For instance, in this case, you could indicate the use of lists tags in HTML helps in presenting a list of information in a well-formed and semantic way.

Reply:
Please use unordered list tags in your header codes, instead of using span tags, this will group a collection of items that do not have a numerical ordering, and their order in the list is meaningless

Reply:
- Great job being polite, phrasing your comment as a suggestion, and giving a reason for the requested change.